test/functional/lua/secure_spec.lua

```diff
@@ -144,6 +152,111 @@ describe('vim.secure', function()
       pcall_err(command, 'write')
       eq(true, api.nvim_get_option_value('readonly', {}))
     end)
+
+    it('directory', function()
+      screen:set_default_attr_ids({
+        [1] = { bold = true, foreground = Screen.colors.Blue1 },
+        [2] = { bold = true, reverse = true },
+        [3] = { bold = true, foreground = Screen.colors.SeaGreen },
+        [4] = { reverse = true },
+      })
+
+      local cwd = fn.getcwd()
+      if #cwd + 23 > 500 then
```

Comment:
```suggestion
      if #cwd + 115 > 500 then
```

Reply:
I must admit I blindly copied this from the test above. What are these magic numbers? Should I also make this change in the previous test?

Reply:
Actually wait. See #33770

Reply:
I've applied your improvement from #33700 to the test I added in this PR. Marking resolved.